src/Components/Home/Home.css

```diff
@@ -0,0 +1,155 @@
+* {
```

Comment:
Even though I really like your design, it's important that we follow the given design as much as possible. It must be really close to being an exact replica 🤓

The design 
![image](https://user-images.githubusercontent.com/15807343/189406199-a2603cf6-861c-43b2-9877-416b3a6b2aae.png)

Please be sure to work on the layout of your design. For example, try mimicking the composition and space between elements.